cogs/smallfunctions.py

```diff
@@ -0,0 +1,43 @@
+import discord
+import random
+import #new file storing priv variables
+from discord.ext import commands
+
+# Class responsible for all the small commands run by server members.
+class SmallFunctions(commands.cog):
+
+# Command function for randomized coinflip. Will return a heads or tails.
+    @commands.command()
+    async def random_coinflip(self, ctx):
+        heads = 1#store in diff file as const variables
+        tails = 0#store in diff file as const variables
+        rand_cf = random.randint(0,1)
+        if (rand_cf == heads):
+            return #gif for heads
+        else:
+            return #gif for tails
+
+
+# Command function decide which side you believe the coin will land on.
+    @commands.command()
+    async def decided_coinflip(self, ctx):
```

Comment:
What is this function for?

Reply:
trying to work on a cf option where you pick which you believe it will land on. Obviously luck based but you try to predict it its more to do with the currency system I'm looking into making.